modules/contact/src/ContactAction.C

```diff
@@ -59,6 +64,13 @@ validParams<ContactAction>()
                              system,
                              "System to use for constraint enforcement.  Options are: " +
                                  system.getRawNames());
+  params.addParam<bool>(
+      "normalize_penalty",
+      false,
+      "Whether to normalize the penalty parameter with the nodal area for penalty contact.");
```

Comment:
I would strike the "for penalty contact" part of this because it is used in both penalty and kinematic enforcement.